### PR TITLE
4749 Add description to all the tests

### PR DESCRIPTION
**Pull request by EvgeniiKlimenko**

### CODE REVIEW

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/FileInputsTests.java [hunk 1]
```diff
@@ -95,47 +95,47 @@ public void counterFileInputTest() {
         counterFileInput.counter().has().text("2 files (62 B in total)");
     }
 
-    @Test
+    @Test(description = "Test checks that file input is dens or not")
```
Comment (MayaElf) [inline jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/FileInputsTests.java:98]: misprint 'densE' :)

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/FileInputsTests.java [hunk 2]
```diff
@@ -95,47 +95,47 @@ public void counterFileInputTest() {
...
-    @Test
+    @Test(description = "Test checks that file size is shown in file input")
     public void showSizeFileInputTest() {
         showSizeFileInput.show();
         showSizeFileInput.uploadFile(pathTXT.toString());
         showSizeFileInput.has().file("TextTestFile.txt (11 B)");
     }
 
-    @Test
+    @Test(description = "Test checks that file inout accept only specific file formats")
```
Comment (MayaElf) [inline jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/FileInputsTests.java:132]: misprint 'inPut'

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/FileInputsTests.java [hunk 3]
```diff
@@ -148,37 +148,37 @@ public void selectionFileInputTest() {
...
 
-    @Test
+    @Test(description = "Test checks that file input has autofocus enabled")
     public void autofocusFileInputTest() {
         backgroundColorFileInput.show();
         backgroundColorFileInput.has().autofocus();
     }
 
-    @Test
+    @Test(description = "Test checks light and drk themes of file inputs")
```
Comment (MayaElf) [inline jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/FileInputsTests.java:165]: misprint 'dArk'

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/FileInputsTests.java [hunk 4]
```diff
@@ -196,90 +196,90 @@ public void messagesFileInputTest() {
...
+    @Test(description = "Test checks that file input field is reversed")
     public void reversedFileInputTest() {
         loadingFileInput.show();
         loadingFileInput.is().reversed();
         denseFileInput.show();
         denseFileInput.is().notReversed();
     }
 
-    @Test
+    @Test(description = "Test checks loading state of file input and it's height")
```
Comment (MayaElf) [inline jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/FileInputsTests.java:247]: misprint 'its' without comma

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/NavigationDrawersTests.java [hunk 5]
```diff
@@ -82,37 +82,37 @@ public void imagesNavigationDrawerTest() {
...
-    @Test
+    @Test(description = "Test checks that drawer is at the right side")
     public void rightNavigationDrawerTest() {
         rightNavigationDrawer.show();
         rightNavigationDrawer.is().right();
     }
 
-    @Test
+    @Test(description = "Test checks overlay drawer has expected parameters")
     public void overlayTemporaryNavigationDrawerTest() {
```
Comment (MayaElf) [inline jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/NavigationDrawersTests.java:116]: it's more about overlay feature, as it has color and opacity :)

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/SelectsTests.java [hunk 6]
```diff
@@ -74,7 +74,7 @@ public void readOnlySelectTest() {
         readonlySelect.is().collapsed();
     }
 
-    @Test
+    @Test(description = "Test checks that only first selected value is shown and others are counted")
     public void selectionSelectTest() {
         selectionSelect.show();
         selectionSelect.select("buzz");
```
Comment (MayaElf) [inline jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/SelectsTests.java:80]: do we have disabled tests in Select? :(

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/composite/SheetsTests.java [hunk 7]
```diff
@@ -25,62 +25,64 @@ public void before() {
...
-    @Test
+    @Test(description = "Test checks that sheet has expected width")
     public void widthSheetsTest() {
         elevationSheets.get(1).show();
         elevationSheets.get(1).has().width(100);
     }
 
-    @Test
+    @Test(description = "Test checks that sheet has outlines")
     public void outlinedSheetsTest() {
```
Comment (MayaElf) [inline jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/composite/SheetsTests.java:68]: 'is outlined' maybe? :)

**Review by MayaElf — CHANGES_REQUESTED** [review]

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/BadgesTests.java [hunk 8]
```diff
@@ -36,62 +36,63 @@ public void typeBadgesTest() {
...
+    @Test(description = "Test checks that badge is inline to icon")
     public void inlineBadgesTest() {
         simpleBadges.get(5).show();
         simpleBadges.get(5).is().inline();
+        simpleBadges.get(6).is().inline();
         simpleBadges.get(4).is().notInline();
     }
 
-    @Test
+    @Test(description = "Test checks that badge is inline to icon")
```
[the remark this answers is not in the document]
Comment (EvgeniiKlimenko) [reply jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/BadgesTests.java:74]: Copy-past :) Changed to correct one.

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/BadgesTests.java [hunk 9]
```diff
@@ -36,62 +36,63 @@ public void typeBadgesTest() {
...
+    @Test(description = "Test checks that badge has text")
     public void textBadgesTest() {
         Badge simpleBadge = simpleBadges.get(1);
         simpleBadge.show();
         simpleBadge.has().text("1");
     }
 
-    @Test
+    @Test(description = "Test checks that badge is inline to icon")
     public void inlineBadgesTest() {
```
[the remark this answers is not in the document]
Comment (EvgeniiKlimenko) [reply jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/BadgesTests.java:67]: Fixed

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/AlertsTests.java [hunk 10]
```diff
@@ -57,61 +58,62 @@ public void heightAlertsTest() {
...
+    @Test(dataProvider = "textAlertsTestsData", dataProviderClass = AlertsTestsDataProvider.class,
+            description = "Test checks text within alert element")
     public void textAlertsTest(int index, String alertText) {
         basicAlerts.get(index).show();
         basicAlerts.get(index).has().text(alertText);
     }
 
-    @Test()
+    @Test(description = "Test checks a dismissible alert")
     public void dismissibleAlertTest() {
```
[the remark this answers is not in the document]
Comment (EvgeniiKlimenko) [reply jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/AlertsTests.java:117]: Looks like I lost focus after dzillion of descriptions.
Fixed!

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/BadgesTests.java [hunk 11]
```diff
@@ -36,62 +36,63 @@ public void typeBadgesTest() {
...
+    @Test(description = "Test checks that badge is bordered or not")
     public void borderedBadgesTest() {
         imageBadge.show();
         imageBadge.has().bordered();
         simpleBadges.get(1).has().notBordered();
     }
 
-    @Test
+    @Test(description = "Test checks that badges is on bottom")
     public void bottomBadgesTest() {
```
[the remark this answers is not in the document]
Comment (EvgeniiKlimenko) [reply jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/BadgesTests.java:47]: Added

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/BadgesTests.java [hunk 12]
```diff
@@ -36,62 +36,63 @@ public void typeBadgesTest() {
...
-    @Test
+    @Test(description = "Test checks that badge is inline to icon")
     public void alignmentBadgesTest() {
         simpleBadges.get(5).show();
         simpleBadges.get(5).has().rightAlignment();
         simpleBadges.get(6).has().leftAlignment();
     }
 
-    @Test
+    @Test(description = "Test checks overlapped badge for button")
```
[the remark this answers is not in the document]
Comment (EvgeniiKlimenko) [reply jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/BadgesTests.java:81]: Fixed.

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/SnackbarsTests.java [hunk 13]
```diff
@@ -112,38 +112,38 @@ public static void roundedSnackbarTest() {
...
                 .and().centered();
     }
 
-    @Test
+    @Test(description = "Test checks that Snackbar is a tile")
     public static void tileSnackbarTest() {
         diffStylesSnackbars.get(4).is().visible()
                 .and().rightAlignment()
                 .and().centered();
     }
```
[the remark this answers is not in the document]
Comment (EvgeniiKlimenko) [reply jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/SnackbarsTests.java:128]: Agree. Changed to a correct one.

[the remark this answers is not in the document]
Comment (EvgeniiKlimenko) [reply jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/SnackbarsTests.java:128]: But name of test is confusing

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/SnackbarsTests.java [hunk 14]
```diff
@@ -95,15 +95,15 @@ public static void timeoutSnackbarTest() {
...
 
-    @Test
+    @Test(description = "Test checks that Snackbar is shaped and has left alignment")
     public static void shapedSnackbarTest() {
         diffStylesSnackbars.get(1).is().visible()
                 .and().shaped()
                 .and().leftAlignment()
                 .and().top();
     }
```
[the remark this answers is not in the document]
Comment (EvgeniiKlimenko) [reply jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/SnackbarsTests.java:104]: Fixed

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/SnackbarsTests.java [hunk 15]
```diff
@@ -112,38 +112,38 @@ public static void roundedSnackbarTest() {
...
 
-    @Test
+    @Test(description = "Test checks that Snackbar is elevated and has left alignment")
     public static void elevationSnackbarTest() {
         diffStylesSnackbars.get(3).is().visible()
                 .and().elevated()
                 .and().leftAlignment()
                 .and().centered();
     }
```
[the remark this answers is not in the document]
Comment (EvgeniiKlimenko) [reply jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/SnackbarsTests.java:121]: Fixed

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/SnackbarsTests.java [hunk 16]
```diff
@@ -112,38 +112,38 @@ public static void roundedSnackbarTest() {
...
 
-    @Test
+    @Test(description = "Test checks that Snackbar has a styled text")
     public static void textSnackbarTest() {
         diffStylesSnackbars.get(5).is().visible()
                 .and().styledText()
                 .and().leftAlignment()
                 .and().bottom();
     }
```
[the remark this answers is not in the document]
Comment (EvgeniiKlimenko) [reply jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/SnackbarsTests.java:136]: Fixed.

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/SwitchesTests.java [hunk 17]
```diff
@@ -152,27 +152,27 @@ public void switchMessagesTest() {
...
+    @Test(description = "Test checks dense of the switch")
     public void denseSwitchTest() {
         variousAttributesSwitches.get(1).show();
         variousAttributesSwitches.get(1).is().dense();
         variousAttributesSwitches.get(5).is().notDense();
     }
 
-    @Test
+    @Test(description = "Test checks that switch is read only and not")
     public void readonlySwitchTest() {
```
[the remark this answers is not in the document]
Comment (EvgeniiKlimenko) [reply jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/SwitchesTests.java:163]: Fixed

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/FileInputsTests.java (hunk 1 above)
Comment (EvgeniiKlimenko) [reply jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/FileInputsTests.java:98]: I wish this misprint should be 'dance'!
Fixed

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/FileInputsTests.java (hunk 2 above)
Comment (EvgeniiKlimenko) [reply jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/FileInputsTests.java:132]: Fixed.

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/FileInputsTests.java (hunk 3 above)
Comment (EvgeniiKlimenko) [reply jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/FileInputsTests.java:165]: Fixed

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/FileInputsTests.java (hunk 4 above)
Comment (EvgeniiKlimenko) [reply jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/FileInputsTests.java:247]: Fixed

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/NavigationDrawersTests.java (hunk 5 above)
Comment (EvgeniiKlimenko) [reply jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/NavigationDrawersTests.java:116]: Yes, but this test is inside in Drawers test. I don't understand - this test is incorrect, or there should be other description about overlay(therefore, this test should be moved to OverlayTests, right?).

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/SelectsTests.java (hunk 6 above)
Comment (EvgeniiKlimenko) [reply jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/SelectsTests.java:80]: You mean appendAndPrependItemSelectTest() ?

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/composite/SheetsTests.java (hunk 7 above)
Comment (EvgeniiKlimenko) [reply jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/composite/SheetsTests.java:68]: Fixed

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/FileInputsTests.java (hunk 1 above)
Comment (MayaElf) [reply jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/FileInputsTests.java]: :) :) :)

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/SnackbarsTests.java (hunk 13 above)
[the remark this answers is not in the document]
Comment (MayaElf) [reply jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/SnackbarsTests.java:128]: Maybe, name is correct? Is this shackbar 'tile'? Can we add 'tile' check in the test?

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/NavigationDrawersTests.java (hunk 5 above)
Comment (MayaElf) [reply jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/NavigationDrawersTests.java:116]: thanks for clarification, everything is ok :)

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/SelectsTests.java (hunk 6 above)
Comment (MayaElf) [reply jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/SelectsTests.java:80]: yes, appendAndPrependItemSelectTest()

**Review by MayaElf — CHANGES_REQUESTED** [review]

**Pull request merged**